### PR TITLE
Clxp 71

### PR DESCRIPTION
**Pull request by adamska27**
https://go1web.atlassian.net/jira/software/projects/CLXP/boards/626?selectedIssue=CLXP-71

**Detailed purpose of the PR**

create certification detail page

**Result and observation**

chromatic link: https://6622682e70ecd8ecf5027fa1-qefbvakvnq.chromatic.com/

- [ ] **Breaking changes ?**  
       If checked, what have you broken ?

- [ ] **Extra lib ?**
      If checked, Which extra lib did you add ? (name, purpose, link ...).

**Testing Strategy**

- [ ] Already covered by tests
- [ ] Manual testing
- [ ] Unit testing

### CODE REVIEW

**Comment by HuyIT9898** [comment]
Hi Adam,  the certification's information block is overflowing, which make the gap in below section might not be as we expected

image

**Comment by HuyIT9898** [comment]
The tex color is mismatch with the icon

image

**Comment by adamska27** [comment]
> The tex color is mismatch with the icon
> 
> image

ah yes, I have fixed that now 👍

**Review by HuyIT9898 — APPROVED** [review]

File: packages/@coorpacademy-components/src/molecule/progress-wrapper/index.js [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+import PropTypes from 'prop-types';
+import {isEmpty, lowerCase, map} from 'lodash/fp';
+import Title from '../../atom/title';
+import ProgressBar from '../progress-bar';
+import {COLORS} from '../../variables/colors';
+import Tag from '../../atom/tag';
+import ButtonLink from '../../atom/button-link';
+import Icon from '../../atom/icon';
+import Provider from '../../atom/provider';
+// eslint-disable-next-line css-modules/no-unused-class
```
[the remark this answers is not in the document]
Comment (adamska27) [reply packages/@coorpacademy-components/src/molecule/progress-wrapper/index.js:11]: as I build my classname dynamically depending on the index like so: ```style[`detailsSection${index}`]```
it's considering that I'm not using the class `detailsSection1` for example present in the style.css file

**Review by AdrieanKhisbe — APPROVED** [review]

**Pull request merged**